applications/luci-app-shadowsocks-libev/luasrc/view/shadowsocks-libev/add_instance.htm

```diff
@@ -1,24 +1,17 @@
 <div class="cbi-section-create cbi-tblsection-create">
-	<br />
-	<div class="table cbi-section-table">
-		<div class="tr cbi-section-table-row">
-			<div class="td cbi-section-table-cell" style="width:140px">
-				<select class="cbi-input-select" id="_newinst.type" name="_newinst.type">
-					<option value="_dummy">-- instance type --</option>
-					<option value="ss_local">ss-local</option>
-					<option value="ss_tunnel">ss-tunnel</option>
-					<option value="ss_redir">ss-redir</option>
-					<option value="ss_server">ss-server</option>
-				</select>
-			</div>
-			<div class="td cbi-section-table-cell" style="width:110px">
-				<input type="text" class="cbi-input-text" id="_newinst.name" name="_newinst.name" placeholder="<%:Name%>"/>
-			</div>
-			<div class="td cbi-section-table-cell left">
-				<input type="submit" class="cbi-button cbi-button-add" name="cbi.cts.<%=self.config%>" value="<%:Add%>" />
-			</div>
-		</div>
+	<div>
+		<select class="cbi-input-select" id="_newinst.type" name="_newinst.type">
+			<option value="_dummy">-- instance type --</option>
+			<option value="ss_local">ss-local</option>
+			<option value="ss_tunnel">ss-tunnel</option>
+			<option value="ss_redir">ss-redir</option>
+			<option value="ss_server">ss-server</option>
+		</select>
 	</div>
+	<div>
+		<input type="text" class="cbi-input-text" id="_newinst.name" name="_newinst.name" placeholder="<%:Name%>"/>
+	</div>
+	<input type="submit" class="cbi-button cbi-button-add" name="cbi.cts.<%=self.config%>" value="<%:Add%>" />
```

Comment:
Hi, the three widgets were expected to be placed on the same row, but now they were stacked closed together on the desktop chrome browser.

Another thing is that the `td` that contains the text "This section contains no values yet" seems to lack a `colspan=N` attribute causing the border-top not spanning the whole row.

<img width="1009" alt="2018-06-22 4 14 20" src="https://user-images.githubusercontent.com/4948057/41766341-4f0b472e-7639-11e8-862d-2d15e6f4a0a9.png">

Reply:
@yousong - thanks fixed

Reply:
@jow- yes, the layout restored but other worms sneak out ;)

The first being the unneeded border-top for the Status line of interface.

<img width="964" alt="2018-06-22 6 08 53" src="https://user-images.githubusercontent.com/4948057/41771442-b8ea7b70-7647-11e8-82a3-ed1b25074c9f.png">

The second being that now the first `td` of an empty now takes up the whole viewpoint that the page is unclickable.

![2018-06-22 6 10 10](https://user-images.githubusercontent.com/4948057/41771447-bbcfef82-7647-11e8-9bfe-bb0b2de3486e.png)

Reply:
Note that the parent tr has "position relative" so the td should be confied to it. Might be some chrome/webkit specific issue though, will check.

Reply:
@yousong - indeed Webkit specific as `display: table-row` Items have undefined behaviour in this context according to spec. Solved it differently now.